NBXplorer/Extensions.cs

```diff
@@ -78,15 +79,7 @@ public static T As<T>(this IActionResult actionResult)
 			}
 			return null;
 		}
-
-		internal static KeyPathInformation AddAddress(this KeyPathInformation keyPathInformation, Network network)
```

Comment:
I don't think you need to refactor this part.

Reply:
Ok